pkg/virt-handler/isolation/process.go

```diff
@@ -17,14 +17,161 @@
  *
  */
 
+/*
+The code in this file has been thankfully taken from the repo https://github.com/mitchellh/go-ps/
+*/
```

Comment:
@alicefr is this OK in terms of adding note for the original repo?

Reply:
Looks OK to me but it might be necessary to add the copyright notice of the MIT license too? I am no lawyer though.

Reply:
Please, add the MIT notice as state in the license
```
The MIT License (MIT)

Copyright (c) 2014 Mitchell Hashimoto

Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

then, this should be good